README.md

```diff
@@ -1,53 +1,172 @@
-# docker-collector-metrics
+# Docker Metrics Collector
 
-docker-collector-metrics is a Docker container that uses Metricbeat to collect metrics and forward those metrics to your Logz.io account.
+To simplify shipping metrics from one or many sources,
+we created Docker Metrics Collector.
+Docker Metrics Collector is a container
+that runs Metricbeat with the modules you enable at runtime.
 
-To use this container, you'll set environment variables in your docker run command.
-docker-collector-metrics uses those environment variables to specify which Metricbeat modules you want to be collected.
-docker-collector-metrics mounts docker.sock to the container itself, allowing Metricbeat to collect the metrics and metadata.
+Docker Metrics Collector ships metrics only.
+If you want to ship logs to Logz.io,
+see [docker-collector-logs](https://github.com/logzio/docker-collector-logs).
 
-docker-collector-metrics ships metrics only. If you want to ship logs to Logz.io, see [docker-collector-logs](https://github.com/logzio/docker-collector-logs).
-
-## docker-collector-metrics setup
+## Configuration
 
 ### 1. Pull the Docker image
 
-Download the logzio/docker-collector-metrics image:
+Download the Docker Metrics Collector image:
 
 ```shell
 docker pull logzio/docker-collector-metrics
 ```
 
 ### 2. Run the container
 
-For a complete list of options, see the parameters below the code block.👇
+For a complete list of options, see the parameters below the code block
+and in the [_Modules_](#modules) section at the bottom of this doc. 👇
 
 ```shell
 docker run --name docker-collector-metrics \
---env LOGZIO_TOKEN="<ACCOUNT-TOKEN>" \
```

Comment:
Hey guys today i have tried to run the aws docker collector myself according to our documentation. it worked great but i have some comments regarding the docs that i thin can improve the experience for our clients.

those parameters are mentioned as “required” but dont appear in the “copy paste” snippet above. i found it a bit confusing:
![chrome-capture (36) (36)](https://user-images.githubusercontent.com/48534558/75878757-c11ca380-5e22-11ea-9429-0e9611e4a5b9.jpg)
![chrome-capture (37) (37)](https://user-images.githubusercontent.com/48534558/75878796-d0035600-5e22-11ea-8ea1-0ba2ca99c85f.jpg)

One last thing, i think it would be useful to mention that if you want to ship metrics from multiple name spaces yo need to separate them by commas in the same line. ( AWS/EC2,AWS/RDS,etc…)

![chrome-capture (38) (38)](https://user-images.githubusercontent.com/48534558/75878844-ead5ca80-5e22-11ea-8436-b554e5d9baf3.jpg)

Reply:
These will be covered in the next few commits after I push. Thanks!